Description: Suppress man page generation and the location of .doctrees
 The sphinx generated man page is not in the right format. Suppress
 generating the man page and including one specifically for Debian in
 the package. In addition, changed the location where the .doctrees are
 getting generated to prevent them from automatically being included in
 the package.
Author: Anuradha Weeraman <anuradha@debian.org>
Last-Update: 2020-10-04
Forwarded: not-needed

--- a/doc/meson.build
+++ b/doc/meson.build
@@ -8,7 +8,7 @@ sphinx_build_dir = meson.current_build_dir()
 
 # Build documentation
 foreach b: doc_builders
-  run_command(sphinx, '-b', b, sphinx_source_dir, join_paths(sphinx_build_dir, b))
+  run_command(sphinx, '-b', b, sphinx_source_dir, join_paths(sphinx_build_dir, b), '-d', join_paths(sphinx_build_dir, '.doctrees'))
 endforeach
 
 # `in` operator requires meson 0.49.0
diff --git a/meson_options.txt b/meson_options.txt
index dc6690b..f0a665c 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -1 +1 @@
-option('doc_builders', type : 'array', choices : ['html', 'man'], value : ['html', 'man'])
+option('doc_builders', type : 'array', choices : ['html', 'man'], value : ['html'])
